public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -174,23 +175,46 @@ export function ServicesContent(props: ServicesProps) {
 
   return (
     <>
-      <SearchBar
-        ref={searchBarRef}
-        query={query}
+      <EuiFlexGroup
+        gutterSize="s"
+        alignItems="center"
+        justifyContent="spaceBetween"
+        style={{ padding: '0 24px' }}
```

Comment:
Same for custom styling here.

Reply:
<img width="1259" alt="Screenshot 2024-08-22 at 1 08 30 PM" src="https://github.com/user-attachments/assets/602b00f5-fa1d-469f-9654-c1124ab13a4a">
Same issue, where using EuiPage to enable padding it ends up generating space.